ocs_ci/ocs/cluster.py

```diff
@@ -611,6 +611,23 @@ def get_rebalance_status(self):
                 and states['count'] == total_pg_count
             )
 
+    @retry(UnexpectedBehaviour, tries=20, delay=10, backoff=1)
+    def get_rebalance_status(self):
```

Comment:
change func name to something like "wait_for_rebalance_to_complete"

Reply:
fixed